main.py

```diff
@@ -283,6 +283,11 @@ def create_circuit(self, request: Request) -> JSONResponse:
         except ValidationError as exception:
             raise HTTPException(400, detail=str(exception)) from exception
 
+        try:
+            self._use_uni_tags(evc)
```

Comment:
Before this validation it's already performing a `evc.sync()`, so shouldn't this be validated before actually saving an EVC?

Reply:
You are right, changed the order